mdn-short-surveys/2023-05-15-security-dx/results.csv

```diff
@@ -0,0 +1,298 @@
+Response ID,Date Submitted,Referer,"Understanding security threats and how they might apply to your web application:To keep your Web app and its users safe from security vulnerabilities, rate how easy or challenging are the following aspects:","Understanding the security model of Web browsers (e.g. same-origin policy, CORS, permissions policy):To keep your Web app and its users safe from security vulnerabilities, rate how easy or challenging are the following aspects:","Keeping up to date the frameworks and libraries your app depends on:To keep your Web app and its users safe from security vulnerabilities, rate how easy or challenging are the following aspects:","Integrating safely third-party services (e.g. login, payment):To keep your Web app and its users safe from security vulnerabilities, rate how easy or challenging are the following aspects:","Configuring the server to match the required security properties (e.g. SSL, HTTP headers):To keep your Web app and its users safe from security vulnerabilities, rate how easy or challenging are the following aspects:","Detecting security vulnerabilities introduced in your development workflow (e.g. cross-site scripting):To keep your Web app and its users safe from security vulnerabilities, rate how easy or challenging are the following aspects:",What are the other main security-related challenges you're facing when developing and deploying Web content?
```

Comment:
The headers are long and hard to grasp since they are built with the pattern "[choice]: [question]". I suggest to drop the latter part.

```suggestion
Response ID,Date Submitted,Referer,"Understanding security threats and how they might apply to your web application","Understanding the security model of Web browsers (e.g. same-origin policy, CORS, permissions policy)","Keeping up to date the frameworks and libraries your app depends on","Integrating safely third-party services (e.g. login, payment)","Configuring the server to match the required security properties (e.g. SSL, HTTP headers)","Detecting security vulnerabilities introduced in your development workflow (e.g. cross-site scripting)",What are the other main security-related challenges you're facing when developing and deploying Web content?
```